src/node.cc

```diff
@@ -1923,6 +1923,63 @@ static void Uptime(const FunctionCallbackInfo<Value>& args) {
 }
 
 
+static void GetResourceUsage(const FunctionCallbackInfo<Value>& args) {
+  Environment* env = Environment::GetCurrent(args);
+
+  uv_rusage_t rusage;
+  int err = uv_getrusage(&rusage);
+  if (err)
+    return env->ThrowUVException(err, "uv_getrusage");
+
+  Local<Object> resource_usage = Object::New(env->isolate());
```

Comment:
I'd prefer this be implemented the same way `fs.Stat` was implemented (https://github.com/iojs/io.js/blob/7dc8eec/lib/fs.js#L105-L134).

@bnoordhuis Do this in another PR, or take care of it in this PR?